charts/osm/values.yaml

```diff
@@ -188,7 +188,7 @@ OpenServiceMesh:
     # -- Toggles Envoy's tracing functionality on/off for all sidecar proxies in the mesh
     enable: false
     # -- Address of the tracing collector service (must contain the namespace). When left empty, this is computed in helper template to "jaeger.<osm-namespace>.svc.cluster.local". Please override for BYO-tracing as documented in tracing.md
```

Comment:
The helper template computes `osm.tracingAddress` as mentioned in the comment. However, I don't believe `osm.tracingAddress` is referenced in any template. Only `.Values.OpenServiceMesh.tracing.address` is referenced in the `preset-mesh-config.yaml`. The above comment could be updated and the helper function removed, or the `preset-mesh-config.yaml` could be updated to reference `osm.tracingAddress`.

Reply:
hmm so previously the computed value from
https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/_helpers.tpl#L6-L10 used to be referenced in osm-configmap.yaml as added with this change: https://github.com/openservicemesh/osm/commit/13ef51563fe9d7b8a36ad3547f96244da8f7faa6

I'm not sure how the osm-config values are set so unsure where/if this is used anymore

Reply:
osm-config was the ConfigMap format of OSM configuration. It was removed in v0.9.0